frontend/packages/aquascope-embed/src/lib.ts

```diff
@@ -23,59 +19,61 @@ import "./styles.scss";
 // be *read only* (because no-interact is true).
 let initEditors = () => {
   // embedded aquascope editors should be <div> tags with the class 'aquascope'
-  document.querySelectorAll<HTMLElement>(".aquascope-embed").forEach(elem => {
-    elem.classList.remove("aquascope-embed");
-    elem.classList.add("aquascope");
+  document
+    .querySelectorAll<HTMLDivElement>(".aquascope-embed")
+    .forEach(elem => {
+      elem.classList.remove("aquascope-embed");
+      elem.classList.add("aquascope");
 
-    // container for the button
-    let btnWrap = document.createElement("div");
-    btnWrap.classList.add("top-right");
+      // container for the button
+      let btnWrap = document.createElement("div");
+      btnWrap.classList.add("top-right");
 
-    // button for computing the receiver permissions
-    let computePermBtn = document.createElement("button");
-    computePermBtn.className = "fa fa-refresh cm-button";
+      // button for computing the receiver permissions
+      let computePermBtn = document.createElement("button");
+      computePermBtn.className = "fa fa-refresh cm-button";
 
-    let initialCode = JSON.parse(elem.dataset.code!);
+      let initialCode = JSON.parse(elem.dataset.code!);
 
-    btnWrap.appendChild(computePermBtn);
-    elem.appendChild(btnWrap);
+      btnWrap.appendChild(computePermBtn);
+      elem.appendChild(btnWrap);
 
-    let serverUrl = elem.dataset.serverUrl
-      ? new URL(elem.dataset.serverUrl)
-      : undefined;
-    let readOnly = elem.dataset.noInteract! == "true";
+      let serverUrl = elem.dataset.serverUrl
+        ? new URL(elem.dataset.serverUrl)
+        : undefined;
+      let readOnly = elem.dataset.noInteract! == "true";
 
-    let ed = new Editor(
-      elem,
-      setup,
-      [receiverPermissionsField.stateField, coarsePermissionDiffs.stateField],
-      err => {
-        if (err.type == "BuildError") console.error(err.error);
-        else if (err.type == "AnalysisError") console.error(err.error);
-        else console.error(err);
-      },
-      initialCode,
-      serverUrl,
-      readOnly
-    );
+      let ed = new Editor(
+        elem,
+        setup,
+        [receiverPermissionsField.stateField],
```

Comment:
We should rename "receiver" throughout the code since we're not just doing receivers any more.